mifospay/src/main/res/values/dimens.xml

```diff
@@ -30,24 +30,28 @@
 
     <dimen name="value_10dp">10dp</dimen>
     <dimen name="value_10sp">10sp</dimen>
+    <dimen name="value_16dp">16dp</dimen>
```

Comment:
Don't declare values like that. All important margins are already declared so you should use these (if not add new margin value with proper name and not VALUE_16DP since it's just useless.

Reply:
@letelete  I don't understand what is the problem in adding a dimen with value 16dp?kindly help me out with this

Reply:
Issues with this solution are:

1. Naming is just completely out-of-logic. What is the sense of having values named like that? It doesn't tell you what is this about, It doesn't tell where it has been used. 

So you could just set hard-coded values in the layout files as well and that would not make any difference. 

You should set names which actually tells something about the value.

We are keeping these ..."value_Xdp" dimens since they were used in the master branch, so to avoid any crashes we keep them here until all layout won't be redesigned.

2. Creating value with 16dp is actually redundant since you probably wanted to set it as a margin. We already have this value for margin and its called: marginItemsInSectionSmall

If this value stand for something else and not margin, you should add a new value but with the PROPER name. Like: `emptyStateImageSize`, because it tells you what is it used for.